app/src/main/java/com/beemdevelopment/aegis/ui/fragments/BehaviorPreferencesFragment.java

```diff
@@ -19,6 +19,12 @@ public void onCreatePreferences(Bundle savedInstanceState, String rootKey) {
 
         _prefs = getPreferences();
 
+        Preference focusSearchPreference = findPreference("pref_focus");
+        focusSearchPreference.setOnPreferenceChangeListener((preference, newValue) -> {
+            getResult().putExtra("needsRefresh", true);
```

Comment:
Triggering a refresh should not be needed here.

Reply:
Forgot to remove it